docs/diagrams/AddCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,41 @@
+@startuml
+hide footbox
+
+actor user
+participant ":Ui" as Ui
+participant ":Parser" as Parser
+participant ":AddCommand" as AddCommand
+
+activate Ui
+user -> Ui : "add..."
+create Parser
+Ui -> Parser : Parser("add...")
+activate Parser
+Ui <-- Parser : parser :Parser
```

Comment:
Maybe you could omit the description here, since it's clear that a Parser object is being constructed? But either way works.